### PR TITLE
Fix: Use originThrowable instead of throwable for Samples

### PR DESCRIPTION
**Pull request by marandaneto**
Merge only when https://github.com/getsentry/sentry-dart/issues/304 fix is deployed

### CODE REVIEW

File: src/includes/configuration/before-send-fingerprint/android.mdx [hunk 1]
```diff
@@ -5,7 +5,7 @@ import java.util.Arrays;
 
 SentryAndroid.init(this, options -> {
   options.setBeforeSend((event, hint) -> {
-    if (event.getThrowable() instanceof SQLiteException) {
+    if (event.getOriginThrowable() instanceof SQLiteException) {
```
Comment (bruno-garcia) [inline src/includes/configuration/before-send-fingerprint/android.mdx:8]: Doesn't the getter of throwable already returns the original throwable?

**Review by bruno-garcia — COMMENTED** [review]
new API?

File: src/includes/configuration/before-send-fingerprint/android.mdx (hunk 1 above)
Comment (marandaneto) [reply src/includes/configuration/before-send-fingerprint/android.mdx:8]: no

**Comment by marandaneto** [comment]
> new API?

wrong docs I'd say

File: src/includes/configuration/before-send-fingerprint/android.mdx (hunk 1 above)
Comment (philipphofmann) [reply src/includes/configuration/before-send-fingerprint/android.mdx:8]: I'm just curious. Why did we rename this?

Comment (marandaneto) [reply src/includes/configuration/before-send-fingerprint/android.mdx:8]: see the linked Dart issue, same thing.

**Review by philipphofmann — APPROVED** [review]
LGTM

**Comment by marandaneto** [comment]
in favor of ttps://github.com/getsentry/sentry-docs/pull/3156

**Pull request closed without merging**